db/migrate/20200213145521_backfill_published_at_for_editions.rb

```diff
@@ -0,0 +1,6 @@
+class BackfillPublishedAtForEditions < ActiveRecord::Migration[6.0]
+  def up
+    Status.where(state: %w[published published_but_needs_2i])
+          .find_each { |s| s.edition.update!(published_at: s.created_at) unless s.edition.published_at }
```

Comment:
This should have ordering on it (as per the trello card) to ensure that it only catches the first publish status of an Edition - they can have multiple based on either clearing a withdrawal or an unreviewed piece of content gaining a review.

Reply:
Ah ha, you're absolutely correct! Sorry I went rogue and thought the ordering was superfluous so removed it, have reverted it back to your suggestion exactly.